core/src/main/scala/cats/Iso.scala

```diff
@@ -0,0 +1,119 @@
+package cats
+
+import cats.Iso.HasIso
+import cats.arrow.{ArrowChoice, Category, Strong}
+import cats.evidence.{=:!=, =~~=}
+import cats.implicits._
+
+trait Iso[->[_, _], A, B] { self =>
```

Comment:
I can change it with another arrow symbol `(-->, ==>, ...?!?)`. I think the <F[_,_]> notation for all classes in the arrow package really encumbers the intuition for the code in those classes. After all it's an arrow really.

Reply:
If uniformity is an issue I can change it in the arrow package also with the new symbol...

Reply:
also reverted <binCompatTest>

Reply:
You want me to change this
```
trait Iso[->[_, _], A, B] { self =>
  def cat: Category[->]
  def to: A -> B
  def from: B -> A

  /** From A <-> B, X <-> Y we can obtain (A, X) <-> (B, Y) if -> has a Strong instance */
  def and[X, Y](that: Iso[->, X, Y])(implicit A: Strong[->]): Iso[->, (A, X), (B, Y)] = ???

  /** From A <-> B, X <-> Y we can obtain Either[A, X] <-> Either[B, Y] if -> has an ArrowChoice instance */
  def or[X, Y](that: Iso[->, X, Y])(implicit A: ArrowChoice[->]): Iso[->, Either[A, X], Either[B, Y]] = ???
}
```
into this ?
```trait Iso[F[_, _], A, B] { self =>
  def cat: Category[F]
  def to: F[A, B]
  def from: F[B, A]

  /** From Iso[F, A, B], Iso[F, X, Y] we can obtain Iso[F, (A, X), (B, Y)] if F has a Strong instance */
  def and[X, Y](that: Iso[F, X, Y])(implicit A: Strong[F]): Iso[F, (A, X), (B, Y)] = ???

  /** From Iso[F, A, B], Iso[F, X, Y] we can obtain Iso[F, Either[A, X], Either[B, Y]] if F has an ArrowChoice instance */
  def or[X, Y](that: Iso[F, X, Y])(implicit A: ArrowChoice[F]): Iso[F, Either[A, X], Either[B, Y]] = ???
}
```
And all because of this ? [https://github.com/typelevel/cats/pull/2845](url)
I think this talk is just starting to stop being productive. I want my code to be understandable and I'm making an active effort for it.
Feel free to do whatever you consider. And thank you for your time.

Reply:
@cosmin33 The latter version is what the rest of the Cats codebase uses, and that's based on many previous conversations and consensus that was developed over time. That doesn't mean we can't change it! That just seems out of the scope of this PR.

Reply:
This has been fun and instructive. There's no point debating a consensus. Good luck with your ceremony, I'll leave you to it then.